dev_guide/builds/jenkins_pipeline_builds.adoc

```diff
@@ -0,0 +1,169 @@
+[[dev-guide-jenkins-pipeline-builds]]
+= Jenkins Pipeline Builds
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[overview]]
+
+== Overview
+
+
+
+[[how-it-works]]
+
+== How It Works
+
+
+[ditaa, "jenkins-pipeline-lifecycle"]
+----
++----------------------------------+
+|                                  |
+|   A Pipeline build is triggered  |
+|   in OpenShift via command line, |--------------------------------+
+|   web console, or API            |                                |
+|                                  |                                |
++----------------------------------+                                |
+                                                                    v
+                                     +------------------------------------------------------------+
+                                     |                                                            |
+                                     |   The Jenkins Master for your project observes that a new  |
+                +--------------------|   Pipeline build object has been created. A new Pipeline   |
+                |                    |   job is then created and a new Jenkins Slave is requested |
+                |                    |                                                            |
+                |                    +------------------------------------------------------------+
+                v
++------------------------------------------------------------+
+|                                                            |
+|   The content of the JenkinsFile is run on the Jenkins     |
+|   Slave pod. A new build on OpenShift is triggered via     |---------------------+
+|   The "Trigger a Build" command in the JenkinsFile         |                     |
+|                                                            |                     |
++------------------------------------------------------------+                     |
+                                                                                   v
+                                                                  +---------------------------------+
+                                                                  |                                 |
+                                                                  |   OpenShift runs the build as   |
+                +-------------------------------------------------|   normal and marks the Pipeline |
+                |                                                 |   build as complete             |
```

Comment:
this description is a bit light.  openshift runs the build as normal, the pipeline logic watches the openshift build for completion before moving on to the next step of the pipeline. (again, this really depends on how your pipeline/jenkinsfile is defined, also)